packages/wasm/pkg/README.md

```
@@ -118,15 +119,40 @@ For contributing or running the full demo app:
 ```bash
 git clone https://github.com/louistrue/ifc-lite.git
 cd ifc-lite
-pnpm install && pnpm dev
+pnpm install
+pnpm build    # Build all packages first
+pnpm dev      # Start the viewer
 ```
 
 Open http://localhost:5173 and load an IFC file.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)